src/xo-app/vm/tab-network.js

```diff
@@ -388,34 +442,17 @@ export default class TabNetwork extends BaseComponent {
       </Row>}
       <Row>
         <Col>
-          {!isEmpty(vm.VIFs)
-            ? <span>
-              <table className='table' style={TABLE_STYLE}>
-                <thead>
-                  <tr>
-                    <th>{_('vifDeviceLabel')}</th>
-                    <th>{_('vifMacLabel')}</th>
-                    <th>{_('vifMtuLabel')}</th>
-                    <th>{_('vifNetworkLabel')}</th>
-                    <th>{_('vifAllowedIps')}</th>
-                    <th>{_('vifStatusLabel')}</th>
-                    <th className='text-xs-right'>{_('vifAction')}</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  {map(vm.VIFs, vif => <VifItem key={vif} vifId={vif} isVmRunning={isVmRunning(vm)} resourceSet={vm.resourceSet} />)}
-                </tbody>
-              </table>
-              {vm.addresses && !isEmpty(vm.addresses)
+          <span>
+            <SortedTable collection={vifs} columns={COLUMNS}
+              individualActions={INDIVIDUAL_ACTIONS} />
+            {vm.addresses && !isEmpty(vm.addresses)
```

Comment:
Indentation.

Reply:
`!isEmpty(vm.addresses)` [should be enough](http://devdocs.io/lodash~4/index#isEmpty).